index.html

```diff
@@ -0,0 +1,131 @@
+          </p>
+        </div>
+        <div class="row comment">
+          <div class="head">
+            <small><strong class="user">Giesche</strong> 25.08.2022 10:30</small>
+          </div>
+          <p>
+            Praesent molestie ante nec metus convallis aliquam. Ut aliquam tincidunt mollis. Maecenas et ex sit amet est
+            vehicula ultrices sed sit amet elit. Suspendisse potenti. Aenean et quam ut purus convallis porttitor.
+            Mauris porttitor pretium elementum. Duis blandit elit tincidunt ipsum ultricies, ut faucibus lorem
+            facilisis. Proin ipsum turpis, pharetra in lorem ac, porta ullamcorper velit. Proin gravida odio eget elit
+            ultricies sodales. Vivamus vel tincidunt ligula. Proin pulvinar pellentesque velit eget luctus. Aliquam
+            vitae enim ut purus vestibulum sollicitudin sit amet eget lacus. Nunc tempus fringilla tincidunt.
+          </p>
+        </div>
+        <div class="row comment">
+          <div class="head">
+            <small><strong class="user">John</strong> 15.08.2022 20:13</small>
+          </div>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin non lorem elementum, accumsan magna sed,
+            faucibus mauris. Nulla pellentesque ante nibh, ac hendrerit ante fermentum sed. Nunc in libero dictum, porta
+            nibh pellentesque, ultrices dolor. Curabitur nunc ipsum, blandit vel aliquam id, aliquam vel velit.
+            Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Sed sit amet mi
+            dignissim, pretium justo non, lacinia libero. Nulla facilisi. Donec id sem velit.
+          </p>
+        </div>
+        <div class="row comment">
+          <div class="head">
+            <small><strong class="user">Susan</strong> 12.08.2022 08:26</small>
+          </div>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin non lorem elementum, accumsan magna sed,
+            faucibus mauris. Nulla pellentesque ante nibh, ac hendrerit ante fermentum sed. Nunc in libero dictum, porta
+            nibh pellentesque, ultrices dolor. Curabitur nunc ipsum, blandit vel aliquam id, aliquam vel velit.
+            Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Sed sit amet mi
+            dignissim, pretium justo non, lacinia libero. Nulla facilisi. Donec id sem velit.
+          </p>
+        </div>
+        <hr />
+      </div>
+    </main>
+    <footer class="my-5 pt-5 text-muted text-center text-small container">
+      <p class="mb-1">© 2022 BeSocial</p>
+      <ul class="list-inline">
+        <li class="list-inline-item"><a href="#">Privacy</a></li>
+        <li class="list-inline-item"><a href="#">Terms</a></li>
+        <li class="list-inline-item"><a href="#">Support</a></li>
+      </ul>
+    </footer>
+    <script src="//code.jquery.com/jquery-1.11.1.min.js"></script>
```

Comment:
```suggestion
```

jquery lite is already included in `boostrap.bundle`. You can remove this

Reply:
When I do this the dropdown to write posts don't work anymore.